manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
Making ```foreman_proxy::plugin``` SCL-aware is tricky since we simply pass the package name, including the prefix. That would be fine if either all plugins are SCL or none, but some is tricky. I think that's why creating a ```$foreman_proxy::scl_plugin_prefix``` variable is a good step in between. That would keep the OS-specific logic out of this file, but allow this file to indicate that SCL should be used if the platform wants it.